src/Enable.Extensions.Configuration/ConfigurationExtensions.cs

```diff
@@ -25,7 +25,8 @@ public static void ApplyDevelopmentOverrides(this IConfiguration configuration)
             Branch.Value,
             BranchPath.Value,
             DatabaseServer.Value,
-            Environment.MachineName);
+            Environment.MachineName.ToLower(),
+            Environment.GetEnvironmentVariable("ENABLEDEV_SERVICEBUSSHAREDACCESSKEY") ?? string.Empty);
```

Comment:
What's setting this then? Is this going to be a one-time setup step for all engineers?